### ISSUE TITLE
RunDialog Button remove/delete not possible

### ISSUE THREAD

**Issue by Roxxinger** [scottdurow/RibbonWorkbench#36]
Hi,

I added a button to run a dialog, with smart buttons. Now i want to delete the button. It is not possible. Tried through ribbon workbench in xrmtoolbox and also through direct solution (customizations.xml) manipulation. The button is always there...I also can set the button to invisible...

Any idea whats causing this?

Br,
Jan

**Comment by scottdurow** [comment]
That is indeed odd - is there a managed layer that contains the button as well? If not and you can’t delete it by removing from the ribbondiffxml manually (which is what the ribbon Workbench does anyway) I think you will need to raise a support ticket with Microsoft.

**Comment by Roxxinger** [comment]
yeah its really weird. How can i check for a managed layer? Don't think there is any. I also can't delete the command...gladly I could use the button for a different command, so thats a workaround but will also raise a ticket with Microsoft.
